doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
Since it only affects the GitHub web UI and not how the final page renders, that doesn't seem like an issue to me...